core/src/main/kotlin/com/strumenta/kolasu/parsing/Parsing.kt

```diff
@@ -82,7 +82,7 @@ data class TokenCategory(val type: String) {
 open class KolasuToken(
     open val category: TokenCategory,
     open val position: Position,
-    open val text: String
+    open val text: String? = null
```

Comment:
Do we want to allow this? Why should a token have no text?

If we keep it this way, I would consider adding a comment to explain under which circumstances `text` can be null

Reply:
When storing them in the LW repo, we don't include the text